openhtf/util/units/__init__.py

```diff
@@ -0,0 +1,1170 @@
+# coding: iso-8859-1
+
+# Copyright 2015 Google Inc. All Rights Reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+"""Units of measure for OpenHTF.
+
+Used to retrieve UNECE unit codes/suffixes as follows:
+
+from openhtf.utils.units import UNITS
```

Comment:
style guide forbids importing symbols from within modules.  you could probably do sys.modules swap and just make the whole module just the `UNITS` dict, but that might break ability to pickle them because the class definition of `Unit` goes away (just a hunch, try it first).  maybe just put the `Unit` definition in `test_record.py` or something and reference it here, then you can do `sys.modules[__name__] = UNITS` at the end and users can do ```Python
from openhtf.util import units

u = units['RADIAN']
```